tslint.json

```diff
@@ -122,7 +122,7 @@
         // TODO: "interface-name": true,
         "jsdoc-format": true,
         "max-classes-per-file": [
-            false, // TODO: true,
+            true, // TODO: true,
```

Comment:
Same comment as before. Remove the "// TODO: true" since you already set it to true